src/components/files-grid-viewer.js

```diff
@@ -43,15 +44,21 @@ function FilesGridViewer({
   };
 
   const renderFile = file => {
-    const Icon = getResourceIcon({ filePath: file.path, isDirectory: file.isDirectory, filled: true });
+    let fileDisplay;
+    if (!file.isDirectory && file.url && getResourceType(file.url) === RESOURCE_TYPE.image) {
```

Comment:
Here you can reduce the condition to `getResourceType(file.url) === RESOURCE_TYPE.image` because it will return `RESOURCE_TYPE.none` for any url you give it that does not end in a file extension (including undefined and null).

Reply:
True, thanks, I removed the second condition.